model/metrics/dotnet/dotnet-aspnetcore.yaml

```diff
@@ -53,6 +58,7 @@ groups:
   - id: metric.aspnetcore.routing.match_attempts
     type: metric
     metric_name: aspnetcore.routing.match_attempts
+    stability: experimental
```

Comment:
Are these still experimental? Aren't they part of .NET 8 now? 🤔

Reply:
see my comment below - will do .NET separately